docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -324,7 +325,7 @@ log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss,SSS} [%t]
 
 **Step 6.** Build the project:
 - To build the project, from the menu choose Build, and then choose Build Project.
-
+* Note that if you run into ```bad option: '-make:transitive'```, please follow the [how to solve scalac-error-bad-option-maketransitive](https://stackoverflow.com/questions/35016945/scalac-error-bad-option-maketransitive-on-mvn-package-via-command-line)
```

Comment:
What change from the stackoverflow page did you apply to solve the error? Can you either apply it directly to the POM in Step 3 so users following the tutorial don't have to worry about it or copy the solution you found directly into the tutorial?